homeassistant/components/amcrest/camera.py

```diff
@@ -82,30 +77,26 @@
 _CBW_BW = "bw"
 _CBW = [_CBW_COLOR, _CBW_AUTO, _CBW_BW]
 
-_SRV_GOTO_SCHEMA = CAMERA_SERVICE_SCHEMA.extend(
-    {vol.Required(_ATTR_PRESET): vol.All(vol.Coerce(int), vol.Range(min=1))}
-)
-_SRV_CBW_SCHEMA = CAMERA_SERVICE_SCHEMA.extend(
-    {vol.Required(_ATTR_COLOR_BW): vol.In(_CBW)}
-)
-_SRV_PTZ_SCHEMA = CAMERA_SERVICE_SCHEMA.extend(
-    {
-        vol.Required(_ATTR_PTZ_MOV): vol.In(_MOV),
-        vol.Optional(_ATTR_PTZ_TT, default=_DEFAULT_TT): cv.small_float,
-    }
-)
+_SRV_GOTO_SCHEMA = {
+    vol.Required(_ATTR_PRESET): vol.All(vol.Coerce(int), vol.Range(min=1))
+}
+_SRV_CBW_SCHEMA = {vol.Required(_ATTR_COLOR_BW): vol.In(_CBW)}
+_SRV_PTZ_SCHEMA = {
+    vol.Required(_ATTR_PTZ_MOV): vol.In(_MOV),
+    vol.Optional(_ATTR_PTZ_TT, default=_DEFAULT_TT): cv.small_float,
+}
 
 CAMERA_SERVICES = {
-    _SRV_EN_REC: (CAMERA_SERVICE_SCHEMA, "async_enable_recording", ()),
-    _SRV_DS_REC: (CAMERA_SERVICE_SCHEMA, "async_disable_recording", ()),
-    _SRV_EN_AUD: (CAMERA_SERVICE_SCHEMA, "async_enable_audio", ()),
-    _SRV_DS_AUD: (CAMERA_SERVICE_SCHEMA, "async_disable_audio", ()),
-    _SRV_EN_MOT_REC: (CAMERA_SERVICE_SCHEMA, "async_enable_motion_recording", ()),
-    _SRV_DS_MOT_REC: (CAMERA_SERVICE_SCHEMA, "async_disable_motion_recording", ()),
+    _SRV_EN_REC: ({}, "async_enable_recording", ()),
```

Comment:
These services aren't registered with the entity service helper so we need to keep the service schema as a voluptuous `Schema` and not just a dict.

Reply:
#49166